src/index.html

```diff
@@ -17,11 +17,399 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.scss"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#moyo"
+        class="header__logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="header__img"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="nav__link is-active"
```

Comment:
According to the checklist, class names that represent the state should be prefixed with 'is-', 'has-', etc. Here 'is-active' should be 'nav__link--is-active' to follow the BEM methodology properly.